apps/test/unit/templates/sectionsRefresh/AdvancedSettingTogglesTest.jsx

```diff
@@ -1,101 +1,73 @@
-import {shallow} from 'enzyme'; // eslint-disable-line no-restricted-imports
+import {render, screen} from '@testing-library/react';
 import React from 'react';
-import sinon from 'sinon'; // eslint-disable-line no-restricted-imports
 
 import AdvancedSettingToggles from '@cdo/apps/templates/sectionsRefresh/AdvancedSettingToggles';
-
-import {expect} from '../../../util/reconfiguredChai'; // eslint-disable-line no-restricted-imports
+import i18n from '@cdo/locale';
 
 describe('AdvancedSettingToggles', () => {
   it('renders PairProgramming and LockThisSection toggles to true and false as default respectively', () => {
-    // Later we might want to set up helper sections to test all of these
-    const wrapper = shallow(
+    render(
       <AdvancedSettingToggles
         updateSection={() => {}}
         section={{pairingAllowed: true, restrictSection: false}}
       />
     );
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-pair-toggle"]').props().isToggledOn
-    ).to.be.true;
-    expect(
-      wrapper.find('ToggleSwitch[id="uitest-lock-toggle"]').props().isToggledOn
-    ).to.be.false;
+    const pairingToggle = screen.getByLabelText(i18n.pairProgramming());
+    expect(pairingToggle).toHaveAttribute('checked');
+    const lockToggle = screen.getByLabelText(i18n.restrictSectionAccess());
+    expect(lockToggle).not.toHaveAttribute('checked');
+    // Check that the other toggles do not appear.
+    const ttsToggle = screen.queryByText(i18n.enableTtsAutoplayToggle());
+    expect(ttsToggle).toBeNull();
```

Comment:
```suggestion
    const ttsToggle = screen.getByText(i18n.enableTtsAutoplayToggle());
```
getByText throws an exception when not found, so we can replace `query` and the `expect` with one `get` call.

Reply:
🤔 if I use `getByText` the test fails because the exception is thrown. I'm failing the example in these [docs](https://testing-library.com/docs/guide-disappearance/#asserting-elements-are-not-present)